bin/cml.js

```diff
@@ -76,9 +76,43 @@ const setupLogger = (opts) => {
   });
 };
 
-const setupTelemetry = async (opts) => {
-  const { cml, cmlCommand: action } = opts;
-  opts.telemetryEvent = await jitsuEventPayload({ action, cml });
+const setupTelemetry = async (opts, { parsed: { defaulted } }) => {
+  const { cml, _: command } = opts;
+  const action = command.join(':');
+
+  const allowedTelemetryOptions = {
+    'runner:launch': {
+      cloud: 'plain',
+      cloudStartupScript: 'masked'
+    },
+    'comment:create': {
+      native: 'plain',
+      publishUrl: 'masked'
+    }
+    // IMPORTANT!  --no-watermark and runner launch --reuse
+  };
+
+  const options = Object.fromEntries(
+    Object.entries(opts)
+      .filter(
+        ([key]) =>
+          Object.prototype.hasOwnProperty.call(
+            allowedTelemetryOptions[action] || {},
+            key
+          ) &&
+          !Object.prototype.hasOwnProperty.call(camelcaseKeys(defaulted), key)
+      )
+      .map(([key, value]) => [
+        key,
+        (allowedTelemetryOptions[action] || {})[key] === 'plain' ? value : '***'
+      ])
+  );
```

Comment:
🤯 
This could be done much more simpler and clear

Reply:
@0x2b3bfa0 doesn't believe in comments :joy:

Reply:
`// It does what you think it does`

Reply:
> _This could be done much more simpler and clear_

Ahem, ahem: https://github.blog/2019-02-14-introducing-draft-pull-requests

Reply:
And note that `Object.prototype.hasOwnProperty.call(object, property)` was a linter whim; I would have used `object.hasOwnProperty(property)`